torch/autograd/__init__.py

```diff
@@ -57,6 +57,13 @@ def backward(tensors, grad_tensors=None, retain_graph=None, create_graph=False,
     This function accumulates gradients in the leaves - you might need to zero
     them before calling it.
 
+    .. note::
+        Using this method with ``create_graph=True`` will create a reference cycle
+        between the parameter and its gradient which can lead to memory leak.
+        We recommend using ``autograd.grad`` below when creating the graph to avoid this.
```

Comment:
"below" seems oddly placed in this sentence. Aren't you recommending it to me here?

Reply:
Yes, I meant the function in the doc just below. But it is unclear, removing it.